### PR TITLE
Address review comments on #148 for JWS

### PR DESCRIPTION
**Pull request by letmaik**
This addresses the comments I made in #148 on the JWS spec.

@shizhMSFT Please review.

### CODE REVIEW

File: signature-envelope-jws.md [hunk 1]
```diff
@@ -52,20 +53,19 @@ An example of the claim is described below
...
-To leverage JWS claims validation functionality already provided by libraries, we have defined `iat`, `exp` as top-level nodes.
+To leverage JWS claims validation functionality already provided by libraries, we have defined `iat`, `exp` as top-level properties.
 
 **ProtectedHeaders**: Notary v2 supports only three protected headers: `alg`, `cty`, and `crit`.
 
 ```jsonc
 {
-    "alg": "RS256",
-    "cty": "application/vnd.cncf.notary.v2.jws.v1",
-    "crit":["cty"]
```
Comment (shizhMSFT) [inline signature-envelope-jws.md:68]: Is there any justification for removing `crit`?

**Review by shizhMSFT — COMMENTED** [review]
Sign-off is required for passing the DCO check.

File: signature-envelope-jws.md (hunk 1 above)
Comment (letmaik) [reply signature-envelope-jws.md:68]: See https://github.com/notaryproject/notaryproject/pull/148#discussion_r857016575

Comment (shizhMSFT) [reply signature-envelope-jws.md:68]: Thanks for reference. Similar stuff is also defined for COSE. /cc @gokarnm

**Review by shizhMSFT — APPROVED** [review]
LGTM. Please sign off the DCO.

File: signature-envelope-jws.md [hunk 2]
```diff
@@ -52,20 +53,19 @@ An example of the claim is described below
...
+    "alg": "PS256",
```
Comment (priteshbandi) [inline signature-envelope-jws.md:67]: We dont support [`RSASSA-PKCS1-v1_5 using SHA-256`](
https://github.com/notaryproject/notaryproject/blob/main/signature-specification.md#signature-algorithm-requirements ).

Usually Its recommended to use PSS over PKCS#1 v1.5  as it's missing a security proof; PSS has a security proof (for the padding mode, not for RSA itself of course). That said, PKCS#1 v1.5 padding for signature generation has not been broken (unlike PKCS#1 v1.5 padding for encryption, which does have vulnerabilities).

PS: [ "alg" (Algorithm) Header Parameter Values for JWS](https://datatracker.ietf.org/doc/html/rfc7518#section-3.1)

**Comment by letmaik** [comment]
Closing this as it was addressed in #158.

**Pull request closed without merging**